server/util/scratchspace/scratchspace.go

```diff
@@ -0,0 +1,60 @@
+package scratchspace
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+)
+
+var (
+	// NOTE: This flag shouldn't be used directly except in the tempDir() func.
+	// We intentionally write to a subdirectory under this dir to avoid blowing
+	// away existing files in the configured directory, in case of accidental
+	// misconfiguration.
+	tempDir_doNotUseDirectly = flag.String("storage.tempdir", os.TempDir(), "Root directory for temporary files. Defaults to the OS-specific temp dir.")
+)
+
+const (
+	// tempSubDir is a subdirectory under the temp dir where we actually write
+	// temporary files.
```

Comment:
nit: if you only have one const you can just write

const tempSubDir = "buildbuddy-scratch"

Re comment can you also mention that we recursively rm everything in this dir when Init() is called?

// tempSubDir is a subdirectory under the temp dir where we actually write
// temporary files. It's contents are recursively removed when Init() is called.

Reply:
Added the comment but kept the const as a section so that future consts added will result in a cleaner diff